ec2/development/tthyer/sc-ec2-linux-jumpcloud-development-tthyer.yaml

```diff
@@ -25,7 +25,7 @@ Metadata:
 Mappings:
   LinuxDistributions:
     Ubuntu:
-      AMIID: ami-0944334364827dc2d
+      AMIID: ami-08030043dfd4f797a  # https://github.com/Sage-Bionetworks/packer-base-ubuntu-bionic/tree/v1.0.5
```

Comment:
my dev template didn't get its AMI updated in the previous PR

Reply:
there's a new release, should use v1.0.6